conbench/templates/benchmark-entity.html

```diff
@@ -65,7 +65,8 @@
             {% for k,v in benchmark.error.items() %}
               <li class="list-group-item" style="overflow-y: auto;">
                 <b>{{ k }}</b>
-                <div align="right" style="display:inline-block; float: right;">{{ v }}</div>
+                <br></br>
+                <div align="left" style="display:inline-block; white-space: pre-line; float: center;">{{ v }}</div>
```

Comment:
```suggestion
                <div align="left" style="display:inline-block; white-space: pre; float: center;">{{ v }}</div>
```

I believe that this will also preserve the tabs/spaces in the strings as well. Could you try it out and see if that's true?

Reply:
You're right. See the image 
![conbench_error_aft2](https://user-images.githubusercontent.com/20308634/160684161-31d1ab80-6ab7-445e-a26f-2bd8cfcd03af.png)

Reply:
Awesome, thanks!